.github/workflows/lint.yml

```
@@ -0,0 +1,53 @@
+name: Lint
+
+on:
+    push:
+        branches: [ main ]
+    pull_request:
+
+jobs:
+    phpcs:
+        name: PHPCS
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 7.4
+                  extensions: posix, dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick
+                  coverage: none
+
+            - name: Install dependencies
+              run: composer install --no-interaction --no-suggest --ignore-platform-reqs
```

@rissajackson - You'll probably want to double check the indentation here, but per @mattstauffer's suggestion, we'll need to add this line _before_ the `composer install` line, like this: 
```suggestion
            - name: Install dependencies
              run: |
                        composer config "http-basic.nova.laravel.com" "${{ secrets.NOVA_USERNAME }}" "${{ secrets.NOVA_PASSWORD }}"
                        composer install --no-interaction --no-suggest --ignore-platform-reqs
```

We'll need to do the same in the `tlint` job below.